### ISSUE TITLE
mnist documentation

### ISSUE THREAD

**Issue by rgentlem** [rafalab/dsbook-part-2#62]
probably should go to dslabs,
but the man page for mnist itself doesn't seem to exist, there is one for mnist_127 - but it is inaccurate, I believe
I see:
> names(mnist_127)
[1] "train" "test" 
> names(mnist_127$train)
[1] "y"   "x_1" "x_2"

but the docs talk about 
mnist_127
Format
An object of class list.

Details
train. A data frame containing training data: labels and predictors.

test. A data frame containing test data: labels and predictors.

index_train. The index of the original mnist training data used for the training set.

index_test. The index of the original mnist test data used for the test set.

- i don't see the index_ vectors...

**Comment by rafalab** [comment]
There is no `mnist` object in dslabs. There is `mnist_27`, `mnist_127` and `read_mnist()`. 

`read_mnist` reads in data from the web. it's way to big to be included in the package.

Am I missing something?
